CHANGELOG.md

```diff
@@ -12,4 +12,5 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Enable CORS Headers
 - Add Multilingual
-- Add default language management with LANGUAGE_CODE
\ No newline at end of file
+- Add default language management with LANGUAGE_CODE
+- - Use the backend's jitsi name to match the JWT token and set the subject to the actual room name.
```

Comment:
Changelog lint job is failing because the line exceeded the max length (80 chars)

```suggestion
- Use the backend's jitsi name to match the JWT token and set the subject to the actual room name.
```